pkg/controller/provider/web/openstack/image.go

```diff
@@ -38,7 +38,7 @@ type Image struct {
 	Visibility                  string            `json:"visibility"`
 	Hidden                      bool              `json:"os_hidden"`
 	Checksum                    string            `json:"checksum"`
-	SizeBytes                   int64             `json:"-"`
+	SizeBytes                   int64             `json:"size"`
```

Comment:
how about documenting what it represents - my previous comment assumed it refers to the actual/apparent size..